tests/Fixtures/Integration/misc/phpdoc_to_comment,phpdoc_var_without_name.test

```diff
@@ -8,19 +8,35 @@ Integration of fixers: phpdoc_to_comment,phpdoc_var_without_name.
 --EXPECT--
```

Comment:
ref https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3911#discussion_r202366775
ref https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3911/files#r208303314

basically, this test got rewritten.

I just tested locally original test against code changes proposed by this PR - the test is failing. This means that the rule behind the fixer got modified, and test that has to be changed (regardless if it's utest or itest) proves that.
I find this to be a BC breaker - if a fixer apply single rule in multiple places (like here - for properties and for variables), and you want to adjust the scope and allow to fix only some of places - it deserves configurable option. If you want to fix only some of places without being able to configure it, it's a BC breaker for the rule.

Now, if I would merge it, nobody would be able to apply the rule for variables.